WordPress/src/main/res/values/strings.xml

```diff
@@ -859,4 +881,39 @@
     <string name="hs__conversation_header">Support chat</string>
     <string name="hs__username_blank_error">Enter a valid name</string>
     <string name="hs__invalid_email_error">Enter a valid email address</string>
+
+    <!-- Available languages -->
+    <string-array name="available_languages" translatable="false">
```

Comment:
Remove `available_languages` from strings.xml now it's generated (and add `available_languages.xml` to the git repo).